alarms/pushbullet_alarm.py

```diff
@@ -22,5 +29,15 @@ def pokemon_alert(self, pkinfo):
 			notification_text = self.name + ": " + notification_text
 		gmaps_link = pkinfo['gmaps_link']
 		time_text =  pkinfo['time_text']
-		push = self.client.push_link(notification_text, gmaps_link, body=time_text)
-	
\ No newline at end of file
+		if hasattr(self, 'channel'):
+			push = self.channel.push_link(notification_text, gmaps_link, body=time_text)
+		else:
+			push = self.client.push_link(notification_text, gmaps_link, body=time_text)
+
+def get_channel(client, channel_tag):
+	req_channel = next((channel for channel in client.channels if channel.channel_tag == channel_tag), None)
+
+	if req_channel is None:
+		raise PushbulletError('No channel found with channel_tag "{}"'.format(channel_tag))
+
+	return req_channel
```

Comment:
Since pushbullet.py version 0.10.0 doesn't have the get_channel function, I had to copy it over from https://github.com/randomchars/pushbullet.py/pull/73/files